### ISSUE TITLE
Illformed gemspec

### ISSUE THREAD

**Issue by elia** [lucasefe/themes_for_rails#52]
I get the following message on version 0.5.0:

```
Invalid gemspec in [/Users/elia/.rvm/gems/ree-1.8.7-2011.03/specifications/themes_for_rails-0.5.0.gemspec]: Illformed requirement ["#<YAML::Syck::DefaultKey:0x105600030> 3.0.11"]
```

**Comment by elia** [comment]
Just noticed that if I rebuild the gem locally (and then install it) I don't get that error anymore…
Perhaps you're using and old rubygems version?

**Comment by lucasefe** [comment]
Try 0.5.1. It fixes problems with 1.8.
